.circleci/config.yml

```diff
@@ -1,295 +1,295 @@
-version: 2.1
-orbs:
-  node: circleci/node@5
+# version: 2.1
```

Comment:
note: Keeping this file here for reference, but will delete before merging the PR